dev/merge_arrow_pr.py

```diff
@@ -122,6 +122,31 @@ def fix_version_from_branch(branch, versions):
                     for project in SUPPORTED_PROJECTS]
 
 
+class ComparableVersion(object):
```

Comment:
This can probably be avoided and just use

```
key=lambda x: (int(_) for _ in x.name.split("."))
```

as tuples can be sorted

Reply:
I don't think apache parquet is maintained separatedly from Arrow anymore. Maybe @kou or @kszucs can confirm that.

Reply:
Please keep the PARQUET project support. We still sometimes use the PARQUET project: https://issues.apache.org/jira/issues/?jql=project%20%3D%20PARQUET%20AND%20status%20%3D%20Open%20AND%20component%20%3D%20parquet-cpp

Reply:
Thanks! I've simplified the solution and supported PARQUET versioning.